namnder/internationella-namnden/body.md

```diff
@@ -1,31 +1,28 @@
 
 Gilla gärna nämndens facebooksida för att få uppdateringar om gruppens evenemang: [Facebooksida](https://www.facebook.com/METAInternationalGroup/?eid=ARDdqkLUe2uRu9PZyhqSOZSHFii_V4D0Pyur3l29_TbwBwnJdzM-PPi6HzHEvk_GjVNlp7DBRUWzpQBj)
 
----
+# Gå med i nämnden
+
+Genom engagemang i nämnden kan organisatörer samla timmar mot de 0.2 bonuspoängen som används för att söka utlandsstudier. Det krävs 25 timmar engagemang under 4 månader för att få bonuspoängen, och det är nämndordföranden (internationella studentkoordinatorn) som har ansvar för att räkna timmarna och registrera dem med THS.
 
-## In English
+Nämnden rekryterar i början av varje termin, då ett antal organisatörer som är taggade att delta i anordnandet av evenemang rekryteras. Nämnden är dock öppen för nya organisatörer året om. Är du intresserad av att gå med i nämnden kan du ansöka om medlemskap (skriv till [isc@datasektionen.se](mailto:isc@datasektionen.se)).
 
-The Computer Science Chapter's International Group!
+## Minska Barriärerna för Internationella Studenter
 
-This is a shared group between the Computer Science Chapter and the Chapter for Media Technology, headed by International Student Coordinator Simon Olander Ålund from CS and International Organizer Simon Sundström from Media.
+Internationella nämnden har ett projekt som heter "Översätta Styrdokument". Projektet syftar till att översätta [viktiga dokument](https://styrdokument.datasektionen.se/) till engelska, så att internationella studenter kan få en bättre förståelse för vad som händer på Datasektionen. Detta är ett projekt som är öppet för alla studenter på Datasektionen. Om du är intresserad av att hjälpa till med detta, vänligen kontakta [isc@datasektionen.se](mailto:isc@datasektionen.se).
 
-The group organizes at least two events per semester for computer science students and media technology students. The group recruits new excited organizers at the beginning of each semester, but is open to new organizers all year round.
+## Evenemang
 
-By helping out in the group organizers can collect hours to count toward their 0.2 GPA bonus to the GPA points that are used to apply for international studies. The group leaders are responsible for registering the hours with THS.
+Internationella nämnden anordnar minst två evenemang per termin för internationella studenter som hör till Konglig Datasektionen eller Sektionen för Medieteknik.
+Exempel på evenemang är internationella gasquer, filmkvällar, brädspelkvällar, karaokekvällar, nätverksevenemang (för att presentera sektionens nämnderna) eller även utomhusaktiviteter som skridskoåkning eller vandring. I början av varje termin samarbetar nämnden även med datas mottagningen (bara HT) och den internationella mottagningen som anordnas av THS (varje termin). 
```

Comment:
Not important but "eller även" sounds weird. Would probably replace it with "och"